AmplifyPlugins/Storage/Podfile.lock

```diff
@@ -12,9 +12,11 @@ PODS:
   - AmplifyTestCommon (1.5.1):
     - Amplify (= 1.5.1)
     - AmplifyTestCommon/AWSPluginsTestCommon (= 1.5.1)
+    - AWSMobileClient (~> 2.20.0)
   - AmplifyTestCommon/AWSPluginsTestCommon (1.5.1):
     - Amplify (= 1.5.1)
     - AWSCore (~> 2.20.0)
+    - AWSMobileClient (~> 2.20.0)
```

Comment:
I'm guessing these are the tests that depend on AWSMobileClient :)?

Reply:
Yes, we should change that and depend on Amplify.Auth.